pkgs/by-name/mi/mixxc/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  pkg-config,
+  libpulseaudio,
+  gtk4-layer-shell,
+  gtk4,
+  libxcb,
+  installShellFiles,
+  enableWayland ? false,
+  enableSass ? false,
+  enableX11 ? false,
```

Comment:
These can all be enabled by default.

Reply:
I realize that is a good idea now that you mention it